### DISCUSSION TITLE
Qual a média de valores vocês costumam cobrar Email Marketing em freelas?

### DISCUSSION THREAD

**Discussion by simaodeveloper** [frontendbr/forum#1031] [Discussão]
Vocês cobram pelo tempo trabalhado com base na seu valor/hora, normalmente já tem valores pré-estabelecidos para esse tipo de projeto, ou tem também o depende, pela complexidade do layout?

OBS: Se conseguirem expor valores, seria legal para debatermos.

**Comment by renanlara** [comment]
Vou contar o que já aconteceu comigo.
Tive contrato PJ com uma agência na qual eu também já fui funcionário e, o meu valor hora com eles era de R$ 35,00. Eu evitava de fazer EMKT justamente pela demora. Alguns podem pensar que isso seria bom pois o retorno em $$$ era bom, mas por outro lado afetava o meu tempo com outros projetos.

Quanto ao valor, cada um é cada um e cada empresa é cada empresa. Nem sempre as agências estão dispostas (vamos dizer assim) a pagar um valor justo pela mão de obra.

Em um outro lado da moeda e com outras agências, já cansei de entregar por necessidade site em WordPress feito em uma semana por R$ 500,00 conto. Como também pelo dobro ou triplo do valor.

Complicado...

**Comment by simaodeveloper** [comment]
@renanlara gostaria de entender mais a fundo o que significa a demora na frase "Eu evitava de fazer EMKT justamente pela demora. ", o que demandava mais tempo na construção dos Email Marketing?

**Comment by felipefialho** [comment]
Acho que não importa o que seja, o ideal é sempre orçar o valor hora.

**Comment by renanlara** [comment]
@simaodeveloper, exatamente. A agência me enviava um PSD com o layout do EMKT, Então, mesmo usando algum template, sempre levava bastante tempo para codar, testar, validar e agendar.

**Comment by adeonir** [comment]
O ideal é mesmo cobrar por hora, até porque fazer EMKT não é tão complexo mas sim demorado.
Nenhuma das técnicas modernas funciona, é tudo em tabelas com css inline.

Mesmo usando um framework como Foundation for Emails, é bem trabalhoso deixar o layout redondo conforme o PSD.
